rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -204,6 +208,35 @@ public void receiveHeaders(BtcBlock[] headers) throws IOException, BlockStoreExc
         }
     }
 
+    /**
+     * Receives only one header of serialized Bitcoin block headers and adds them to the internal BlockChain structure.
+     * @param header The bitcoin headers
+     */
+    public Integer receiveHeader(BtcBlock header) throws IOException, BlockStoreException {
+
+        long diffTimeStamp = bridgeConstants.getTimeReceiveHeadersBetweenCalls();
+
+        long actualTimeStamp = System.currentTimeMillis();
```

Comment:
We can't use system time during vm execution

Reply:
Ahh ok. Thank you! I will change it.